data-pipeline/Dockerfile

```diff
@@ -28,7 +35,9 @@ RUN pip install uv
 
 COPY pyproject.toml .
 COPY requirements/ ./requirements/
-RUN uv pip install --system -r ./requirements/requirements.txt --no-cache-dir
+# if sufficiently motivated, we could be clever enough to keep the image size down by
```

Comment:
I don't love this--GitHub actions provides a CI env variable that could, in theory, be used as a way to only include test files and dependencies when we're building docker in a CI context. But the test assets are lightweight enough that I didn't bother doing that.

Reply:
it seems good to always have these anyways, right? then we can dev against the docker image?

Reply:
Agree, I'm over-thinking it because these dependencies likely don't increase the size of the Docker image enough to really worry about it.

In instances where the size does matter, you'll often see a pattern where the image is built w/o dev dependencies by default. Then for local dev and CI runs, the docker `build` command is passed a flag that instructs docker to use `requirements-dev` instead.